runtime/altair/src/weights/pallet_uniques.rs

```diff
@@ -185,4 +185,22 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+
+	fn buy_item() -> Weight {
```

Comment:
Need input here how to weigh things or what value is the right one! Currently it's `44_219_000` but need advice on how to weigh functions @branan @mustermeiszer

Reply:
So `buy_item` is a new extrinsic from unqiues?

These files here are automatically generated. For making this compilable just put a zero weight. E.g. `Weight::from_ref_time(0)`. We can go through running the benchmarks in the cloud or locally tomorrow. This will then generate the correct values for `buy_item`